src/search/pdbs/rcg.cc

```diff
@@ -0,0 +1,87 @@
+#include "rcg.h"
+
+#include "../option_parser.h"
+
+#include "../task_proxy.h"
+
+#include "../utils/countdown_timer.h"
+#include "../utils/logging.h"
+#include "../utils/rng.h"
+
+#include <algorithm>
+#include <unordered_set>
+
+using namespace std;
+
+namespace pdbs{
+static bool time_limit_reached(
+    const utils::CountdownTimer &timer, utils::Verbosity verbosity) {
+    if (timer.is_expired()) {
+        if (verbosity >= utils::Verbosity::NORMAL) {
+            utils::g_log << "time limit reached." << endl;
+        }
+        return true;
+    }
+    return false;
+}
+
+Pattern generate_pattern_rcg(
```

Comment:
What does RCG stand for? Is this something I should be familiar with? If not, is it small enough that we can use the non-abbreviated version in the file or class name?